bigtable_input.py

```diff
@@ -156,6 +162,30 @@ def _game_keys_as_array(ds):
     return make_single_array(ds)
 
 
+def delete_rows(args):
+    """Delete the given row keys from the given Bigtable.
+
+    The args are (table_path, row_keys), but are passed as a single
+    argument in order to work with multiprocessing.Pool.map .
+    """
+    table_path, row_keys = args
+    # Unpack project, instance, and name from path.
+    (project_name,
+     instance_name,
+     table_name) = _unpack_table_path.match(table_path).groups()
+    bt_table = bigtable.Client(
```

Comment:
does bigtable.Client really take its args like this?... *facepalm*

Reply:
Yeah, the `Client` is constructed with a project, from which the caller looks up an instance, from which the caller looks up a table. The unpacking of the args from the path is the part that I can change, per the earlier comment, by simply keeping those three strings in the instance.

Reply:
Used a named tuple to store the Bigtable spec.